packages/dso-toolkit/libs/_bootstrap.scss

```diff
@@ -64,7 +64,7 @@
 @import "bootstrap/component-animations";
 // @import "bootstrap/dropdowns"; Applicable styling moved to @dso-toolkit/styling
 // @import "bootstrap/input-groups"; -> Deleted, deprecated
-@import "bootstrap/navs";
+// @import "bootstrap/navs";
```

Comment:
Ik begrijp niet helemaal wat ik hier lees: De `@import` is weg, maar de file bestaat nog? En heeft styling?

Reply:
Ah ja, dit was een test. Laten we er smaen naar kijken: ik vermoed namelijk dat hij helemaal weg kan.

Reply:
We hebben 'm weer aangezet 🎉